AirCasting/ABConnector/BluetoothManager.swift

```diff
@@ -49,9 +49,12 @@ class BluetoothManager: NSObject, BluetoothCommunicator, ObservableObject {
     @Published var centralManagerState: CBManagerState = .unknown
     @Published var mobileSessionReconnected = false
     var observed: NSKeyValueObservation?
+    @Published var isReconnectionOn = false
+    @Published var didAlreayCancellConnection = false
```

Comment:
I'm not sure about the logic with this variable, because it is used in different functions with the assumption of a specific flow, but okay. I don't like the name of this variable. What do you think about `connectingTimedOut`?

Reply:
Typo is fixed, thanks. In addition to the name - I think it appropriately says what it does and what is its purpose. If you will force the change of name I will do it, but another way I would like it to stay

Reply:
Okay, I will not force anything, but I have to say I don't understand the name. Who already cancelled connection? User? Device?

Reply:
That's bad, I will come up with something new and place it here so, please let me know then what you think

Reply:
What would you say for `peripheralConnectionIsCancelled`?